test/test_torch.py

```diff
@@ -15339,6 +15368,12 @@ def inner(self, device, dtype):
     ('div', '', _small_3d, lambda t, d: [_number(3.14, 3, t)], 1e-1),
     ('div', 'tensor', _small_3d,
         lambda t, d: [_small_3d(t, d, has_zeros=False)], 1e-1),
+    # Note: precision for floor_divide is 1 since a small (1e-5, for example)
+    # error in division can lead to an difference of 1 post-truncation
+    # (e.g. .9999 vs 1 post truncation is 0 vs 1)
+    ('floor_divide', '', _small_3d, lambda t, d: [_number(3.14, 3, t)], 1, 1, 1, _types),
+    ('floor_divide', 'tensor', _small_3d,
+        lambda t, d: [_small_3d(t, d, has_zeros=False)], 1, 1, 1, _types),
```

Comment:
what are these actually comparing?  CPU vs CUDA results?  Those should be the same, right?

Reply:
These tests do compare CPU and CUDA results. *should* is correct. If you expect division on the CPU, CUDA, and TPU to be bitwise equivalent then you wouldn't have an issue. In practice we could set the precision to our standard 1e-5 and it probably would never bother us, either. But if division isn't bitwise-equivalent and the error is such that it could change 0.9999 to 1., for example, then the test would fail when that occurred.

Reply:
I would optimistically assume they are bitwise equivalent and if things start failing back out our assumption.

Reply:
Updated the precision. Turns out we still need a 1 for torch.half.